app/uk/gov/hmrc/channelpreferences/connectors/OutboundProxyConnector.scala

```diff
@@ -141,10 +141,10 @@ object OutboundProxyConnector {
   private def processResponseHeaders(headers: Seq[HttpHeader]): Map[String, String] =
     expandToMap(headers).filter(_._1 != CONTENT_TYPE)
 
-  private def fullPath(request: Request[_]): String =
+  def fullPath(request: Request[_]): String =
     if (request.rawQueryString.nonEmpty) s"${request.path}?${request.rawQueryString}" else request.path
 
-  private def loggedHeaders(headers: Seq[HttpHeader]): Map[String, String] =
+  def loggedHeaders(headers: Seq[HttpHeader]): Map[String, String] =
```

Comment:
if this method is public just to be accessible - it could be opened to the package `private[connectors]`

Reply:
yes, updated now